### ISSUE TITLE
Exported SGF file is unreadable

### ISSUE THREAD

**Issue by Mbr18** [CamWagner/lizzie#434]
When an SGF is exported Lizzie produces a unreadable SGF file (attachments: 01/02). Sabaki: This file is unreadable. When the first three comments are removed Lizzie produces a readable SGF file (attachments: 03/04).

[SGF files.zip](https://github.com/featurecat/lizzie/files/2569601/SGF.files.zip)

**Comment by fohristiwhirl** [comment]
I can open the file in a different SGF editor, I wonder if it's a Sabaki issue...

**Comment by fohristiwhirl** [comment]
Ah, it's the ] character in the comment not being escaped properly.

**Comment by zsalch** [comment]
Yes, the Lizzie does not append the escaped char for the ']' char in the comment.
Maybe need to fix.

**Comment by featurecat** [comment]
Is this now working?
